airbyte-cdk/python/airbyte_cdk/sources/declarative/manifest_declarative_source.py

```diff
@@ -119,6 +123,7 @@ def _configure_logger_level(self, logger: logging.Logger):
             logger.setLevel(logging.DEBUG)
 
     def _validate_source(self):
+        # Validates the connector manifest against the schema auto-generated from the low-code backend
         full_config = {}
         if "version" in self._source_config:
```

Comment:
do we still need these checks now that version and all are part of the schema?

Reply:
As a sanity check yes. I think we want to at least leave the old validator and the new one to ensure parity between the two for now. But yes we would remove this later